### ISSUE TITLE
Pulling CentOS 7 will actually pull CentOS 6.x

### ISSUE THREAD

**Issue by thefosk** [CentOS/CentOS-Dockerfiles#33]
```
docker run -ti centos:7 /bin/bash
[root@ffff21388003 /]# cat /etc/redhat-release
CentOS release 6.7 (Final)
```

I am pretty sure a month ago or so it was working fine.

**Comment by jperrin** [comment]
This appears to be a caching issue on the docker side. You can see from  https://github.com/docker-library/official-images/blob/master/library/centos that the hashes for 7, and latest are the same, and so a pull should fetch the same images. I also saw what you did, 

```
info: Success!
[jperrin@lappy ~]$ docker run -ti centos:7 /bin/bash
Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.
[root@2067d7e9a515 /]# cat /etc/redhat-release 
CentOS release 6.7 (Final)
```

However a docker pull centos:7 claims an update, though nothing new was downloaded. This means it used an existing layer I already had ( centos:latest)

```
[jperrin@lappy ~]$ docker pull centos:7
7: Pulling from docker.io/centos
47d44cb6f252: Already exists 
f6f39725d938: Already exists 
f9a8cbc8dd13: Already exists 
f37e6a610a37: Already exists 
0f73ae75014f: Already exists 
docker.io/centos:7: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
Digest: sha256:c28c6a9260a3d82609c20f27c6654f3a1e80f167392021386c7f33901a27d08d
Status: Downloaded newer image for docker.io/centos:7
```

Following that update, it works now

```
[jperrin@lappy ~]$ docker run -ti centos:7 /bin/bash
Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.
[root@f2155964b30c /]# cat /etc/centos-release
CentOS Linux release 7.1.1503 (Core) 
```

In short, do a fresh pull and the problem should go away, however this does raise an interesting issue to keep an eye on with docker....
